cranelift/filetests/filetests/isa/aarch64/condbr.clif

```diff
@@ -149,53 +149,6 @@ block0(v0: i128, v1: i128):
 ;   csel x0, x7, x10, eq
 ;   ret
 
-function %f(i64, i64) -> i64 {
-block0(v0: i64, v1: i64):
-  v2 = ifcmp v0, v1
```

Comment:
Rather than deleting, can we convert these tests to use `icmp` / `brnz` pairs, so we're still testing the proper matching of the two together (and assuring that we don't accidentally materialize the 0/1 value into a register)?

Reply:
Yep, that's a good call!